.github/scripts/build.js

```diff
@@ -57,33 +65,39 @@ async function getCommandFromComment({ core, context, github }) {
   let command = "none";
   const trimmedFirstLine = commentFirstLine.trim();
   if (trimmedFirstLine[0] === "/") {
-    switch (trimmedFirstLine) {
+    const parts = trimmedFirstLine.split(' ').filter(p=>p !== '');
+    const commandText = parts[0];
+    switch (commandText) {
       case "/test":
         {
-          if (gotNonDocChanges) {
-            command = "run-tests";
-            const message = `:runner: Running tests: https://github.com/${repoFullName}/actions/runs/${runId} (with refid \`${prRefId}\`)`;
-            await addActionComment({ github }, repoOwner, repoName, prNumber, commentUsername, commentLink, message);
-          } else {
+          // Docs only changes don't run tests with secrets so can run regardless of whether
+          if (!gotNonDocChanges) {
```

Comment:
I know this isn't introduced in your PR, but I struggle a little with the name of this variable.

You're basically testing "is this a doc-only change". Could the variable be called `docOnlyChange` or `nonCodeChange`?

This is a nit. Feel free to ignore.

Reply:
Yeah, I agree - updated

Reply:
oops - realised that I forgot to push this commit. Can pick this up in any future changes in the code!